example-runner/pom.xml

```diff
@@ -49,7 +49,7 @@
     </scm>
 
     <properties>
-        <!-- Dockerfile-maven-plugin configurable base image - openjdk:11.0-jre-slim -->
+        <!-- Dockerfile-maven-plugin configurable base image - openjdk:11.0 -->
```

Comment:
Not enough information to tell the reader which Docker image this is.

Reply:
still open comment